### PR TITLE
Fixes deprecation warning regarding using `includes` over `contains`

### PR DESCRIPTION
**Pull request by twokul**
screen shot 2016-09-22 at 8 58 09 am

### CODE REVIEW

File: config/ember-try.js [hunk 1]
```diff
@@ -2,12 +2,6 @@
 module.exports = {
   scenarios: [
     {
-      name: 'default',
```
Comment (twokul) [inline config/ember-try.js]: not sure what this was for since tests are running against `release`, `beta` and `canary`

Comment (ronco) [inline config/ember-try.js]: Good point, probably don't need that.  Can you update `.travis.yml` to eliminate that test run as well?

Comment (twokul) [inline config/ember-try.js]: will do

Comment (twokul) [inline config/ember-try.js]: @ronco done!

**Comment by twokul** [comment]
@ronco could you publish a new version?

**Comment by ronco** [comment]
@twokul Published as 3.0.1

**Comment by twokul** [comment]
@ronco thanks!

**Comment by simenheg** [comment]
It should be noted that this breaks compatibility with all Ember versions below 2.8.

A backward-compatible solution would be to use the [ember-runtime-enumerable-includes-polyfill
](https://github.com/rwjblue/ember-runtime-enumerable-includes-polyfill) package.

**Comment by ronco** [comment]
Thanks for pointing that out @simenheg .  I definitely dropped the ball on the semver compliance there.  I'll try to get a version with the polyfill out soon.

**Pull request merged**